src/main/java/org/ibp/api/java/germplasm/GermplasmListService.java

```diff
@@ -66,4 +66,6 @@ void addGermplasmEntriesToList(
 
 	void addGermplasmListEntriesToAnotherList(String cropName, String programUUID, Integer destinationListId, Integer sourceListId,
```

Comment:
Possible cleanup: Check if we can remove `addGermplasmListData` from service (seems to be used internally in GermplasmServiceImpl class and in test only)